### ISSUE TITLE
Create a `development` branch

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/refinebio-docs#161]
We expect we will need to write user-facing documentation long before implementing and releasing some new refine.bio features. We should create a `development` branch to facilitate this. We'll have to look into Read The Docs behavior – if we can select what branch to serve the docs from, we can probably make `development` the _default branch_ on GitHub.

The scope of this issue also includes updating the README to include this information.

**Comment by jaclyn-taroni** [comment]
I expect that this line controls which branch we use: https://github.com/AlexsLemonade/refinebio-docs/blob/c8ad916008a3331fcc61f45fc452d0f6a559ad9f/docs/conf.py#L184

**Comment by davidsmejia** [comment]
> I expect that this line controls which branch we use:
> 
> https://github.com/AlexsLemonade/refinebio-docs/blob/c8ad916008a3331fcc61f45fc452d0f6a559ad9f/docs/conf.py#L184

This line is for building absolute paths to the file. 

I have enabled PR preview branches for this project on readthedocs.org. 
Also under Verrsions, I activated development as `Active` and `Hidden` which means that it will build when pushed to. It will also be omitted from search results.

After builds are working again I think we can close this.
